src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -220,6 +222,44 @@
 
             return alloyEditor;
         }
+
+        countWordsCharacters(container, editorHtml) {
+            let counterWrapper = container.parentElement.getElementsByClassName('ez-character-counter');
+
+            if (counterWrapper.length) {
+                let wordCharacterWrappers = counterWrapper[0].getElementsByTagName('span');
+                let characterCount = 0;
+                let wordCount = 0;
+                let countableTags = this.fetchCountableTags(editorHtml);
+
+                countableTags.forEach((tag) => {
+                    let sanitizedText = this.cleanWhiteCharacters(tag.innerText);
+
+                    wordCount += sanitizedText ? sanitizedText.split(' ').length : 0;
+                    characterCount += sanitizedText.length;
+                });
+
+                wordCharacterWrappers[0].innerText = wordCount;
```

Comment:
You can't rely on the order of getting element. What if someone will add another span before yours?